.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
-dist/*.min.js
+demo/dist/*.js
```

Comment:
demo/dist/*?

Reply:
Can't be unfortunatley as we need the `index.html`, but later on we could copy it from the src directory at build.